pkg/filesharecontroller/selector/filter_test.go

```diff
@@ -0,0 +1,398 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
may not need this file if we are using controller/selector/filter

Reply:
It is to test filter, it is needed as we have to do unit testing and as it uses fileshare proto buf we  have to test for file share feature.